Dockerfile

```diff
@@ -16,4 +16,5 @@ WORKDIR /refugerestrooms
 COPY Gemfile /refugerestrooms/Gemfile
 COPY Gemfile.lock /refugerestrooms/Gemfile.lock
 RUN bundle install
+RUN apt-get install -y chromium
```

Comment:
I put this at the end instead of with all the `apt-get` statements so Docker can use its cache for everything that came before.